### PR TITLE
Profanity 0.7.0

### PR DESCRIPTION
**Pull request by toon23**
###### Things done

* changed the repo owner to [profanity-im](https://github.com/profanity-im), because https://github.com/boothj5/profanity redirects to to this
* bump version to 0.7.0
* added optional omemoSupport (enabled by default)

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/profanity/default.nix [hunk 1]
```diff
@@ -8,25 +8,27 @@
...
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "profanity-${version}";
-  version = "0.6.0";
+  version = "0.7.0";
 
   src = fetchFromGitHub {
-    owner = "boothj5";
+    owner = "profanity-im";
```
Comment (aanderse) [inline pkgs/applications/networking/instant-messengers/profanity/default.nix]: Please explain why this change can be trusted, preferably citing an official source.

**Review by aanderse — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/instant-messengers/profanity/default.nix (hunk 1 above)
Comment (toon23) [reply pkgs/applications/networking/instant-messengers/profanity/default.nix]: I've just noticed that the link https://github.com/boothj5/profanity redirects to https://github.com/profanity-im/profanity, so I'm assumed that it's better to use this. I'm a new user of profanity since it has OMEMO-support, and I cannot find an official resource announcing that change. If the redirect is not enough to trust this change I will ask on mailing list for a official statement.

Comment (aanderse) [reply pkgs/applications/networking/instant-messengers/profanity/default.nix]: @toon23 No need, that is sufficient. I just like to encourage people to provide that information in their PRs. Thanks :smile:

**Comment by aanderse** [comment]
@GrahamcOfBorg build profanity

**Comment by aanderse** [comment]
@marsam If @toon23 rebases on the latest master (see https://github.com/NixOS/nixpkgs/pull/66154) will `profanity` build on `darwin`?

> ../Libraries/libcar/Sources/Rendition.cpp:29:10: fatal error: 'compression.h' file not found
#include <compression.h>

**Comment by toon23** [comment]
@aanderse: I've rebased the branch to current master.

**Review by aanderse — APPROVED** [review]

**Review by aanderse — CHANGES_REQUESTED** [review]
Hmm... sorry, just noticed the commit messages. Please change messages to be:
- profanity: 0.6.0 -> 0.7.0
- profanity: Add optional omemo support (enabled by default)

**Comment by toon23** [comment]
@aanderse: I changed the commit messages. Thank you for your patience. :smiley:

**Comment by aanderse** [comment]
No problem at all. Thanks for your contribution! :tada:
